content/en/docs/tasks/security/automtls/index.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Automatic mutual TLS
+description: A simplified workflow to adopt mutual TLS with minimal configuration overhead.
+weight: 10
+keywords: [security,mtls,ux]
+---
+
+This tasks shows a simplified workflow for mutual TLS adoption[authentication overview](/docs/concepts/security/#authentication).
+
+With Istio auto mutual TLS feature, you can adopt mutual TLS by only configuring Authentication Policy without worrying about destination rule.
+
+Istio tracks the server workloads are migrated to Istio sidecar, and configure client sidecar to send mutual TLS traffic to those workloads automatically, and send plain text traffic to workloads
+without sidecars. This allows you to adopt Istio mutual TLS incrementally with minimal manual configuration.
+
+## Before you begin
+
+* Understand Istio [authentication policy](/docs/concepts/security/#authentication-policies) and related
+[mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication) concepts.
+
+* Have a Kubernetes cluster with Istio installed (e.g use `install/kubernetes/istio-demo.yaml` as described in
+[installation steps](/docs/setup/install/kubernetes/#installation-steps), set `global.mtls.enabled` to false  and `global.mtls.auto` to true using [Helm](/docs/setup/install/helm/)). For example,
+
+{{< text bash >}}
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system  \
+  --set global.mtls.enabled=false  --set global.mtls.auto=true \
+  --values install/kubernetes/helm/istio/values-istio-demo.yaml | kubectl apply -f -
+{{< /text >}}
+
+## Instructions
+
+### Setup
+
+Our examples use three namespaces, `foo`, `mixed`, and `legacy`. 
```

Comment:
Why do you need all these namespaces? Seems to me that this task doesn't do anything with namespaces, they are just confusing the message. All you need for this task is a client and two httpbin services (one with sidecar and one without)? Am I missing something?

Reply:
You're right. I want to illustrate the idea that  auto mTLS works for all the cases when people adopt istio

- server all have sidecars
- server partially have sidecars
- strict or permissive or plain text for the authn policy

Maybe I should emulate a CUJ of migrating workloads to Istio? Or creating different deployments (inline yaml?) at the same time?

The only issue is that we already have a `mtls-migration` task page, which only covers `PERMISSIVE` case, not auto mTLS.

What do you think?

Reply:
offline discussion, we still use namespace, but

- sleep only in legacy and full
- more explaination on delta behavior of the auto mtls.